lib/vmpooler/api/v1.rb

```diff
@@ -200,6 +200,21 @@ def update_pool_template(payload)
       result
     end
 
+    def reset_pool(payload)
+      result = { 'ok' => false }
+
+      pool_index = pool_index(pools)
+      pools_reset = 0
+
+      payload.each do |poolname, count|
+        backend.sadd('vmpooler__poolreset', poolname)
+        pools_reset += 1
```

Comment:
this `pools_reset` is not used

Reply:
maybe increase a metric instead?

Reply:
Re-reading this it looks like we can just remove it. With poolsize and template it is possible a change is not made depending on the data you supply. With poolreset, it always resets the pool upon request. So, how many of the requested pools were reset is a moot point.